### PR TITLE
Finished tic tac toe

### PR DESCRIPTION
**Pull request by AlexanderRusak**
added game loop for endless play

### CODE REVIEW

File: HW6/HW2 TicTacToe.cpp [hunk 1]
```diff
@@ -105,7 +105,8 @@ void moveAI(string array[][SIDE])
 {
 
 	Sleep(1500);
-	srand(time(NULL));
+
```
Comment (AWolfenstein) [inline HW6/HW2 TicTacToe.cpp:124]: не нужный пробел

File: HW6/HW2 TicTacToe.cpp [hunk 2]
```diff
@@ -8,34 +8,39 @@
...
+void createPlayField(char* array[][4]);
+void showPlayField(char*  array[][4]);
+void makeMove(char*  array[][4]);
+void showMessage(char* message);
+bool isEmptyField(char*  array[][4], int move);
+void moveAI(char*  array[][4]);
+bool isWin(char*  array[][4]);
+bool isMatchHorizontal(char*  array[][4]);
+bool isMatchVertical(char*  array[][4]);
+bool isMatchDiagonal(char*  array[][4]);
```
Comment (VadimAlkhimenok) [inline HW6/HW2 TicTacToe.cpp:20]: Wht do you use *?

File: HW6/HW2 TicTacToe.cpp [hunk 3]
```diff
@@ -94,22 +108,25 @@ void makeMove(string array[][SIDE])
...
 
 	Sleep(1500);
-	srand(time(NULL));
+	
+   	srand(time(NULL));
 	
 	int move = rand() % (SIDE * SIDE) + 1;
 	int moveCounter = 0;
-	string zero = "|0|";
+	char* zero = "|0|";
```
Comment (VadimAlkhimenok) [inline HW6/HW2 TicTacToe.cpp:129]: define

File: HW6/HW2 TicTacToe.cpp [hunk 4]
```diff
@@ -128,11 +145,11 @@ void moveAI(string array[][SIDE])
 	}
 }
 
-void showMessage(string message)
+void showMessage(char* message)
 {
 	cout << message;
 }
-bool isEmptyField(string array[][SIDE], int move)
+bool isEmptyField(char*  array[][4], int move)
```
Comment (VadimAlkhimenok) [inline HW6/HW2 TicTacToe.cpp:152]: why 4? explain it

File: HW6/HW2 TicTacToe.cpp [hunk 5]
```diff
@@ -148,14 +165,14 @@ bool isEmptyField(string array[][SIDE], int move)
 	}
 	return true;
 }
-bool isWin(string array[][SIDE])
+bool isWin(char* array[][4])
```
Comment (VadimAlkhimenok) [inline HW6/HW2 TicTacToe.cpp:168]: the same

File: HW6/HW2 TicTacToe.cpp [hunk 6]
```diff
@@ -271,9 +288,13 @@ int game(){
 	{
```
Comment (VadimAlkhimenok) [inline HW6/HW2 TicTacToe.cpp:289]: delete

**Review by VadimAlkhimenok — COMMENTED** [review]
Hi! I left some comments. Fix them

**Pull request merged**